---a/net/minecraft/network/play/INetHandlerPlayServer.java
+++b/net/minecraft/network/play/INetHandlerPlayServer.java
@@ -25,8 +25,8 @@
 import net.minecraft.network.play.client.C18PacketSpectate;
 import net.minecraft.network.play.client.C19PacketResourcePackStatus;
 
-public interface INetHandlerPlayServer extends INetHandler
-{
+public interface INetHandlerPlayServer extends INetHandler {
+
     void handleAnimation(C0APacketAnimation packetIn);
 
     void processChatMessage(C01PacketChatMessage packetIn);
@@ -72,4 +72,4 @@
     void handleSpectate(C18PacketSpectate packetIn);
 
     void handleResourcePackStatus(C19PacketResourcePackStatus packetIn);
-}
+}
